stories/Modal.story.jsx

```diff
@@ -1,22 +1,109 @@
+/* eslint-disable no-alert */
+
 import React, { useState } from "react";
 import styled from "styled-components";
 import { storiesOf } from "@storybook/react";
 import { Button, Colors, Row, Col, Modal } from "../src/index";
 
 const LayoutStyles = styled.div`
   height: 150vh;
-  .green {
+  .green-bg {
     height: 25%;
     background: ${Colors.brandPrimary};
   }
-  .white {
+  .white-bg {
     height: 25%;
     background: #f5f5f5;
   }
 `;
 
 storiesOf("Modal", module)
   .add("basic", () => <Modal>A very basic modal.</Modal>)
+  .add(
+    "only Submit button",
+    () => {
+      const ModalUseCase = () => {
+        const [open, toggleModal] = useState(true);
+
+        return (
+          <>
+            <Button onClick={() => toggleModal(true)}>
+              Launch composed modal
+            </Button>
+            {open && (
+              <Modal
+                onClose={() => toggleModal(false)}
+                onSubmit={() => alert("I am submitting!")}
+                submitText="Submit"
+              >
+                A very basic modal that wont scroll the background when opened.
+              </Modal>
+            )}
+          </>
+        );
+      };
+      return <ModalUseCase />;
+    },
+    {
+      info:
+        "Sumbit button is show since property 'onSubmit' is being passed on the component."
```

Comment:
```suggestion
        "Sumbit button is shown since property 'onSubmit' is being passed to the component."
```

Reply:
Not sure how I missed this the first time but:
```suggestion
        "Submit button is show since property 'onSubmit' is being passed on the component."
```

Reply:
@paschalidi this was marked as resolved without being changed?

Reply:
interesting. I missed that sorry.